modules/ROOT/pages/api-led-deploy.adoc

```diff
@@ -23,7 +23,7 @@ image:deploy1.png[dialog for deploying to CloudHub]
 . Enter `hello-world-greeting-firstname-lastname`; every app and API name must be unique across CloudHub. This example uses `hello-world-greeting-sandy-kim`.
```

Comment:
Lets update 5.1.2 image since the deploy pop up looks a bit different

Reply:
Image has been updated